R/assertions.R

```diff
@@ -303,9 +303,14 @@ assert_character_vector <- function(arg, values = NULL, named = FALSE,
 #' If set to `FALSE` and `arg` is `NULL` then an error is thrown. Otherwise,
 #' `NULL` is considered as valid value.
 #' @param arg_name string indicating the label/symbol of the object being checked.
+#'
+#' When the argument for `arg_name` can be input as a quote using
+#' `enexpr()`, i.e. `assert_filter_cond`, `assert_symbol`, and `assert_expr`, then
+#' it is specified without `enexpr()` using `gsub()`.
```

Comment:
This looks confusing to me. What is the intention? Do we still need it after moving the `gsub()` call to the default value?

Reply:
@bundfussr Sorry, was trying to find a way to word it and add the change to the default value so its documented for the user. I can remove it.

Reply:
I see. It may be helpful for users which are not so familiar with regular expressions.

I would add it only to the three functions where we use `gsub()` in the default value and rephrase the text. What about the following?
> By default the expression specified for `arg` is used. If it is of the form `enexpr(<argument name>)` , the `enexprs()` part is removed. For example if `arg = enexpr(filter_add)` is specified, `arg_name` defaults to `"filter_add"`.